README.rst

```diff
@@ -119,9 +119,22 @@ The example code assumes you've set the following environment variables:
     response = metric_client.send_batch((temperature, errors, summary))
     response.raise_for_status()
     print("Sent metrics successfully!")
+    
+Find and use data
+-----------------
+
+Tips on how to find and query your data in New Relic:
+
+* `Find metric data <https://docs.newrelic.com/docs/data-ingest-apis/get-data-new-relic/metric-api/introduction-metric-api#find-data>`_
+* `Find trace/span data <https://docs.newrelic.com/docs/understand-dependencies/distributed-tracing/trace-api/introduction-trace-api#view-data>`_
+
+For general querying information, see:
+
+* `Query New Relic data <https://docs.newrelic.com/docs/using-new-relic/data/understand-data/query-new-relic-data>`_
+* `Intro to NRQL <https://docs.newrelic.com/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql>`_
 
 Limitations
 -----------
-The New Relic Telemetry APIs are rate limited. Please reference the documentation for `New Relic Metrics API <https://docs.newrelic.com/docs/introduction-new-relic-metric-api>`_ and `New Relic Trace API Requirements and Limits <https://docs.newrelic.com/docs/apm/distributed-tracing/trace-api/trace-api-general-requirements-limits>`_ on the specifics of the rate limits.
+The New Relic Telemetry SDKs are rate limited. Please reference the documentation for `New Relic Metrics API <https://docs.newrelic.com/docs/introduction-new-relic-metric-api>`_ and `New Relic Trace API requirements and limits <https://docs.newrelic.com/docs/apm/distributed-tracing/trace-api/trace-api-general-requirements-limits>`_ on the specifics of the rate limits.
```

Comment:
I think the statement "The New Relic Telemetry APIs are rate limited" is accurate. The SDKs themselves do not rate limit requests. The rate limiting is controlled by the New Relic service backing the APIs.
```suggestion
The New Relic Telemetry APIs are rate limited. Please reference the documentation for `New Relic Metrics API <https://docs.newrelic.com/docs/introduction-new-relic-metric-api>`_ and `New Relic Trace API requirements and limits <https://docs.newrelic.com/docs/apm/distributed-tracing/trace-api/trace-api-general-requirements-limits>`_ on the specifics of the rate limits.
```

Reply:
no problem makes sense. I wasn't sure of that. Mainly seemed weird to call them the Telemetry APIs; I'd never seen them referred to that way.